### PR TITLE
Groups actions some more commits split from https://github.com/cockpit-project/cockpit/pull/18136

### PR DESCRIPTION
**Pull request by KKoukiou**

### CODE REVIEW

File: pkg/users/accounts-list.js [hunk 1]
```diff
@@ -295,31 +346,31 @@ const GroupsList = ({ groups, accounts }) => {
                                         </Label>
                                     );
                                 })}
-                        <Button key="more" className="group-more-btn" isInline variant='link' onClick={() => setIsExpanded(!isExpanded)}>
-                            {cockpit.format(_("$0 more..."), groups.length - 3)}
-                        </Button>
+                        {filtered_groups.length > 3 && <Button key="more" className="group-more-btn" isInline variant='link' onClick={() => setIsExpanded(!isExpanded)}>
```
Comment (cockpituous) [inline pkg/users/accounts-list.js:349]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18284-20230202-142732-3b71f187-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/users/local.js [hunk 2]
```diff
@@ -87,8 +88,16 @@ function AccountsPage() {
     if (groupsExtraInfo.length == 0 || accountsInfo.length == 0) {
         return <EmptyStatePanel loading />;
     } else if (path.length === 0) {
-        return <AccountsMain accountsInfo={accountsInfo} current_user={current_user_info && current_user_info.name} groups={groupsExtraInfo || []} />;
-    } else {
+        return (
+            <AccountsMain
+                accountsInfo={accountsInfo}
+                current_user={current_user_info && current_user_info.name}
+                groups={groupsExtraInfo || []}
```
Comment (cockpituous) [inline pkg/users/local.js:95]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18284-20230202-142732-3b71f187-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/users/accounts-list.js (hunk 1 above)
Comment (cockpituous) [inline pkg/users/accounts-list.js:349]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18284-20230202-161842-791100a2-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/users/local.js (hunk 2 above)
Comment (cockpituous) [inline pkg/users/local.js:95]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18284-20230202-161842-791100a2-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/lib/cockpit-components-dialog.jsx [hunk 3]
```diff
@@ -315,7 +315,7 @@ export function show_modal_dialog(props, footerProps) {
             dialogObj.footerProps.dialog_done = closeCallback;
         }
         dialogObj.footerProps.set_error = e => {
-            error = e;
+            error = e === 'object' && e != null ? (e.message || e.toString()) : e;
```
Comment (cockpituous) [inline pkg/lib/cockpit-components-dialog.jsx:318]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18284-20230202-204107-84adbdf7-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/users/local.js (hunk 2 above)
Comment (cockpituous) [inline pkg/users/local.js:95]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18284-20230202-204107-84adbdf7-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/lib/cockpit-components-dialog.jsx (hunk 3 above)
Comment (cockpituous) [inline pkg/lib/cockpit-components-dialog.jsx:318]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18284-20230203-083113-4255a28d-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/users/accounts-list.js (hunk 1 above)
Comment (cockpituous) [inline pkg/users/accounts-list.js:349]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18284-20230203-083113-4255a28d-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/users/local.js (hunk 2 above)
Comment (cockpituous) [inline pkg/users/local.js:95]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18284-20230203-083113-4255a28d-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/lib/cockpit-components-dialog.jsx (hunk 3 above)
Comment (cockpituous) [inline pkg/lib/cockpit-components-dialog.jsx:318]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18284-20230203-100012-e8856962-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/users/accounts-list.js (hunk 1 above)
Comment (cockpituous) [inline pkg/users/accounts-list.js:349]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18284-20230203-100012-e8856962-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/users/local.js [hunk 4]
```diff
@@ -87,26 +88,31 @@ function AccountsPage() {
...
```
Comment (cockpituous) [inline pkg/users/local.js:95]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18284-20230203-100012-e8856962-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/lib/cockpit-components-dialog.jsx (hunk 3 above)
Comment (cockpituous) [inline pkg/lib/cockpit-components-dialog.jsx:318]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18284-20230203-133703-75368b81-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/users/accounts-list.js (hunk 1 above)
Comment (cockpituous) [inline pkg/users/accounts-list.js:349]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18284-20230203-133703-75368b81-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/users/local.js [hunk 5]
```diff
@@ -76,37 +77,45 @@ function AccountsPage() {
...
```
Comment (cockpituous) [inline pkg/users/local.js:98]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18284-20230203-133703-75368b81-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/lib/cockpit-components-dialog.jsx (hunk 3 above)
Comment (cockpituous) [inline pkg/lib/cockpit-components-dialog.jsx:318]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18284-20230203-153811-4fa2b4cc-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/users/accounts-list.js (hunk 1 above)
Comment (cockpituous) [inline pkg/users/accounts-list.js:349]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18284-20230203-153811-4fa2b4cc-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

File: pkg/users/local.js (hunk 4 above)
Comment (cockpituous) [inline pkg/users/local.js:95]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18284-20230203-153811-4fa2b4cc-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)

**Review by jelly — APPROVED** [review]

**Comment by jelly** [comment]
```
# 1 TEST PASSED [65s on 4-ci-srv-06]
ok 333 test/verify/check-users TestAccounts.testBasic [ND@5] # TODO still fails with python bridge

```

We can now enable it, I left it out so we can land this PR first.

**Pull request merged**